test/e2e_main_test.ts

```diff
@@ -24,8 +23,7 @@ describe('toClosureJS', () => {
     if (!closure) {
       diagnostics.forEach(v => console.log(JSON.stringify(v)));
       assert.fail();
-      // TODO(lucassloan): remove when the .d.ts has the correct types
-      return {compiledJS: '', sourceMap: new SourceMapConsumer('' as any)};
+      return;
```

Comment:
`noImplicitReturns`.  In JS if a function doesn't return anything it implicitly returns undefined, in the sense that if you write `let x = foo();` then x ends up `undefined` if `foo` doesn't return anything.

The error was:
```
test/e2e_main_test.ts(15,66): error TS7030: Not all code paths return a value.
```
before, the compiler was assuming you wanted to return this object here and then 'undefined' below.